### ISSUE TITLE
Cypher Import - Node labels 

### ISSUE THREAD

**Issue by caligolaz** [jexp/neo4j-shell-tools#42]
Hi Jexp, thank you so much for your great work !
I'm using your lighting fast tool to import big amount of date and it really rocks!!

Now i'm trying to import labels of nodes via CSV, something like that:

create(:Node: {label-from-a-csv-field} {field1:{field1}})

Is it possible ?
Best

**Comment by jexp** [comment]
With

#{label-from-a-csv-field}

Sent from mobile device

Am 29.08.2014 um 13:01 schrieb caligolaz notifications@github.com:

**Comment by caligolaz** [comment]
Great, thanx!

**Comment by FelikZ** [comment]
@jexp is it possible to specify an array of labels as it done in default `neo4j-import` tool?

"multiple;labels;another_one"

**Comment by jexp** [comment]
Yep separated with the array separator, which is comma by default

**Comment by FelikZ** [comment]
@jexp is there a hidden option to change this separator? I am only see `-d` option which is not designed for that.

Thanks

**Comment by jexp** [comment]
Oh sorry, wrong repo I thought this was batch-import

No I don't think there is currently an option for that in here
